[2023-08-22 11:03:19,083][ignite.engine.engine.Engine][INFO] - Engine run starting with max_epochs=1.
[2023-08-22 11:03:20,418][ignite.engine.engine.Engine][ERROR] - Current run is terminating due to exception: shape '[64, 32, 8, 64]' is invalid for input of size 16908288
[2023-08-22 11:03:20,447][ignite.engine.engine.Engine][ERROR] - Engine run is terminating due to exception: shape '[64, 32, 8, 64]' is invalid for input of size 16908288
